app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -0,0 +1,64 @@
+package fr.free.nrw.commons.feedback;
+
+import fr.free.nrw.commons.feedback.model.Feedback;
+
+/**
+ * Creates a wikimedia recognizable format
+ * from feedback information
+ */
+public class FeedbackContentCreator {
+    private StringBuilder stringBuilder;
+    private Feedback feedback;
+
+    public FeedbackContentCreator(Feedback feedback) {
+        this.feedback = feedback;
+        init();
+    }
+
+    /**
+     * Initializes the string buffer object to append content from feedback object
+     */
+    public void init() {
+        // Localization is not needed here, because this ends up on a page where developers read the feedback, so English is the most convenient.
+
+        stringBuilder = new StringBuilder();
+        stringBuilder.append("== ").append("Feedback for version").append(feedback.getVersion()).append(" ==");
+        stringBuilder.append("\n");
+        stringBuilder.append(feedback.getTitle());
+        stringBuilder.append("\n");
+        stringBuilder.append("\n");
+        if (feedback.getApiLevel() != null) {
+            stringBuilder.append("* ").append("API level: ").append(feedback.getApiLevel());
+            stringBuilder.append("\n");
+        }
+        if (feedback.getAndroidVersion() != null) {
+            stringBuilder.append("* ").append("Android version: ")
+                .append(feedback.getAndroidVersion());
+            stringBuilder.append("\n");
+        }
+        if (feedback.getDeviceManufacturer() != null) {
+            stringBuilder.append("* ").append("Device manufacturer: ")
```

Comment:
How about reusing the String `device_manufacturer` found in `strings.xml` rather than duplicate it? It will be easier to maintain when we decide to reword a section. Idem for other sections. Thanks!

Reply:
@nicolas-raoul I think we need to keep it in English that's why I am not calling it from string.xml

Reply:
Yes, always using the English string is OK here. But we can dynamically get the English string from strings.xml since it is already present there for the UI. That avoids duplicating the strings.